app/views/dataset.scala.html

```diff
@@ -4,8 +4,8 @@
 
 @import helper._
 
-@main("", "Dataset: lobid-gnd") { 
-	<h3>Daten<small><a title="Beschreibung als JSON-LD anzeigen" href='@routes.HomeController.dataset("json")'><img class='json-ld-icon' src='@routes.Assets.versioned("images/json-ld.png")'></a></small></h3>
+@main("", "Dataset: RPPD") { 
```

Comment:
Hmm, weird, I get `<title>Dataset: lobid-gnd</title>` for https://rppd.lobid.org/dataset#! . Can this be a caching problem? (I do not find it anywhere in thecode.)